package.json

```diff
@@ -27,8 +27,6 @@
         "d2-manifest": "1.0.0",
         "eslint-plugin-no-relative-import-paths": "^1.5.3",
         "font-awesome": "4.7.0",
-        "lodash": "4.17.21",
-        "lodash.product": "18.10.0",
         "purify-ts": "1.2.0",
```

Comment:
After a "yarn install" I get changes in yarn.lock, I guess it was not committed?